doc/articles/getting-started/wizard/extensions.md

```diff
@@ -0,0 +1,72 @@
+
+    ![](Assets/localization.jpg)
+    
+    The default settings comes with the following pre-set languages: English (*en*), Spanish (*es*), French (*fr*), and Potuguese - Brazil (*pt-BR*).
+
+    Uno.Extensions.Localization expands [Microsoft.Extensions.Localization](https://learn.microsoft.com/en-us/dotnet/core/extensions/localization) to all OSs supported by Uno Platform.  
+    Read the full Localization documentation [here](xref:Overview.Localization) or learn more about [Globalization](https://learn.microsoft.com/en-us/dotnet/core/extensions/globalization).
+
+- **Navigation**
+  - **Regions**
+    Navigation is an integral part of any app and for that, Uno Platform has developed a navigation system that is registered and served using DI and is fully integrated with the navigation specific UI controls out-there, such as `Frame`, `NavigationView`, and `TabBar`.  
+    It is a region-based navigation. A Region is the abstraction of the view responsible for handling navigation. Regions are structured into a logical hierarchical representation that shadows the navigation-aware controls in the visual hierarchy. The hierarchy allows navigation requests to be propagated up to parent and down to child regions as required.  
+    This library is also responsible for showing pop-ups, flyouts, dialogs, and other interactions with the user.  
+    Learn more about Uno's Navigation library [here](xref:Overview.Navigation).
+
+  - **Blank**  
+    Provides the default [WinUI frame navigation](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.frame).
+
+- **Logging**  
+    Logging is a crucial component in an app that enables the developer to emit log messages whenever an important action or execution is taken by the app. This then lets you trace back any errors or issues that may arise in the future.
+    There are several Logging tools out there, with one of the most common of them being [Microosft.Extensions.Logging](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging).  
+    Uno Platform provides its Uno.Extensions.Logging to bring Logging to all platforms it supports.      
+    Uno.Extensions.Logging is covered in more detail [here](xref:Overview.Logging).
+
+  - **Console**  
+  Generates an app configured to write debug-level logging information to the Console.
+
+  - **Default**  
+  Generates an app that uses Dependency Injection to configure logging that writes output to Console.
+
+  - **Serilog**  
+  Generates an app that uses Dependency Injection to configure logging with [Serilog](https://github.com/serilog/serilog).
+
+  Refer to the [Logging](xref:Overview.Logging) documentation for more information.
+
+- **Authentication**  
```

Comment:
Why is this here?

Reply:
I thought it's worth showing it here as well as this qualifies as an extension.

Reply:
Ok removed